### ISSUE TITLE
Expo compatibility

### ISSUE THREAD

**Issue by EduardoRicardez** [infoxicator/react-native-star-prnt#68]
Hi,
I dont find in the docs about expo compatibility, 
can i use it with expo?

**Comment by infoxicator** [comment]
Haven't tried this for a while so I am unsure of Expo compatibility

**Comment by hunwalk** [comment]
Nope, it will never be compatible with expo unless you eject, or create your own build server.
Expo has a standard set of libraries which is compiled together with the app. This lib contains Native Modules which means this probably wont be included in the final build, even though you might be able to reach the js part of the lib. (but the bluetooth connection part is defined in native modules)
